client/driver/rkt_test.go

```diff
@@ -335,12 +339,14 @@ func TestRktDriver_UserGroup(t *testing.T) {
 	require := assert.New(t)
 
 	task := &structs.Task{
-		Name:   "etcd",
+		Name:   "sleepy",
 		Driver: "rkt",
 		User:   "nobody",
 		Config: map[string]interface{}{
-			"image": "docker://redis:3.2",
-			"group": "nogroup",
+			"image":   "docker://redis:4-alpine",
```

Comment:
Should we just use the image sha to ensure we’re consistently testing with the same image?

Reply:
I will drop the redis version back to 3.2. Switching to 4 seems like it's inviting unwanted image churn.